test/upgrade.test.js

```diff
@@ -1,32 +1,32 @@
 'use strict'
 
-const { test, skip } = require('tap')
+const { test } = require('node:test')
 const Fastify = require('..')
 const { connect } = require('node:net')
 const { once } = require('node:events')
 const dns = require('node:dns').promises
 
-async function setup () {
+async function setup (t) {
   const localAddresses = await dns.lookup('localhost', { all: true })
   if (localAddresses.length === 1) {
-    skip('requires both IPv4 and IPv6')
+    t.skip('requires both IPv4 and IPv6')
```

Comment:
`t` is undefined here.

I think the `setup` function should become a `test()` and then:
https://nodejs.org/api/test.html#contextskipmessage

Reply:
I updated the test see commit here:  35e8351